sweepai/utils/openai_proxy.py

```diff
@@ -0,0 +1,27 @@
+import os
+import openai
+
+class OpenAIProxy:
+    OPENAI_API_KEY_GPT35 = os.environ.get("OPENAI_API_KEY_GPT35")
```

Comment:
don't get these as envvars, they are passed in like this:

openai_proxy = OpenAIProxy(
    OPENAI_API_KEY_GPT35,
    OPENAI_API_KEY_GPT4,
    OPENAI_API_KEY_GPT4_32K,
    OPENAI_FALLBACK
)